src/useAnimate.ts

```diff
@@ -20,7 +20,10 @@ export default function useAnimate(
     duration = 0.3,
     easeType = 'linear',
   } = props;
-  const transition = `all ${duration}s ${easeType} ${delay}s`;
+  const transition = React.useMemo(
```

Comment:
Nice

Reply:
I've updated PR (moved one condition to setTimeout), I thought that `isPlaying` should inform whether animation is ended or not (with that approach we can easily unmount component when animation finished like in react-transition-group)

Reply:
actually, I looked this again, this react `usememo` doesn't add too much performance value here.

Reply:
It depends, for simple animation its about 3 calls per one `play`, but when you ie. change other props after first render it will be more and more